packages/react/src/SearchControl/SearchControl.tsx

```diff
@@ -577,15 +577,19 @@ function getPage(search: SearchRequest): number {
 
 function getTotalPages(search: SearchRequest, lastResult: Bundle): number {
   const pageSize = search.count ?? DEFAULT_SEARCH_COUNT;
-  return Math.ceil(getTotal(search, lastResult) / pageSize);
+  const total = getTotal(search, lastResult);
+  return total > 0 ? Math.ceil(total / pageSize) : 1;
```

Comment:
What is the change here?  It seems like the only time the return value would be different is if `total === 0`, before it would return `0`, now it will return `1`?  Is that desired?

Reply:
Confirmed that this is not desired - i had a misunderstanding about how Mantine `Pagination` handles 0 pages

Reply:
Will revert